cuppa/src/main/java/org/forgerock/cuppa/Configuration.java

```diff
@@ -77,4 +79,19 @@ public void setAdditionalReporter(Reporter reporter) {
     public Options getRunOptions() {
         return runOptions;
     }
+
+    /**
+     * Remove the test transforms that are defined by the core cuppa framework. If this method is called, the
+     * {@link ConfigurationProvider} will need to make sure to add the following filters, if it intends to support the
+     * features they provide:
+     * <ul>
+     *     <li>{@link org.forgerock.cuppa.transforms.ExpressionTagTestBlockFilter}</li>
+     *     <li>{@link org.forgerock.cuppa.transforms.TagTestBlockFilter}</li>
+     *     <li>{@link org.forgerock.cuppa.internal.filters.OnlyTestBlockFilter}</li>
+     *     <li>{@link org.forgerock.cuppa.internal.filters.EmptyTestBlockFilter}</li>
+     * </ul>
+     */
+    public void removeCoreTestTransforms() {
```

Comment:
What about `withoutCoreTransforms` which returns a new `Configuration` instance?

Reply:
To do that would still involve a change to the public API of `ConfigurationProvider`, so I would prefer not to...